blockchain/tx_pool.go

```diff
@@ -312,16 +312,21 @@ func (pool *TxPool) loop() {
 			// Handle inactive account transaction eviction
 		case <-evict.C:
 			pool.mu.Lock()
-			for addr := range pool.queue {
+			for addr, beat := range pool.beats {
 				// Skip local transactions from the eviction mechanism
 				if pool.config.KeepLocals && pool.locals.contains(addr) {
+					delete(pool.beats, addr)
```

Comment:
It this line required?

Reply:
If these is not this line, the local account's beat can not be removed.

Reply:
I understand. Thank you!